test/intl402/DurationFormat/constructor/constructor/locales-invalid.js

```diff
@@ -0,0 +1,17 @@
+// Copyright 2022 Igalia, S.L. All rights reserved.
```

Comment:
Looking at the structure of existing tests elsewhere in Intl, e.g. for DateTimeFormat, it seems like these tests should go directly in the `test/intl402/DurationFormat` directory, without the `constructor/constructor/`. See the thread https://github.com/tc39/test262/pull/3406#discussion_r809537160